CMakeLists.txt

```diff
@@ -0,0 +1,178 @@
+# Copyright 2018 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 2.8.12)  # Ships with Ubuntu 14.04 (Trusty)
+
+project(async_grpc)
+
+set(ASYNC_GRPC_MAJOR_VERSION 1)
```

Comment:
Start with a lower version number because it should only increase in the future. (Otherwise you install libasync_grpc.so.1.0 which is misleading)

Reply:
Done